src/main/java/ua/edu/internship/interview/data/enumeration/InterviewStatus.java

```diff
@@ -0,0 +1,27 @@
+package ua.edu.internship.interview.data.enumeration;
+
+import ua.edu.internship.interview.service.utils.exceptions.InvalidStateTransitionException;
+
+import java.util.EnumSet;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.EnumSet'
<!-- __reviewdog__:ChAyMjY4YWFmZTU3YmMyNWFjEgpjaGVja3N0eWxl -->